build/Targets/Packages.props

```diff
@@ -163,83 +163,20 @@
-    <SystemLinqParallelVersion>4.3.0</SystemLinqParallelVersion>
-    <SystemNetHttpVersion>4.3.3</SystemNetHttpVersion>
-    <SystemNetNameResolutionVersion>4.3.0</SystemNetNameResolutionVersion>
-    <SystemNetPrimitivesVersion>4.3.0</SystemNetPrimitivesVersion>
-    <SystemNetSecurityVersion>4.3.0</SystemNetSecurityVersion>
-    <SystemNetSocketsVersion>4.3.0</SystemNetSocketsVersion>
-    <SystemObjectModelVersion>4.3.0</SystemObjectModelVersion>
-    <SystemReflectionVersion>4.3.0</SystemReflectionVersion>
-    <SystemReflectionExtensionsVersion>4.3.0</SystemReflectionExtensionsVersion>
     <SystemReflectionMetadataVersion>1.6.0</SystemReflectionMetadataVersion>
-    <SystemReflectionPrimitivesVersion>4.3.0</SystemReflectionPrimitivesVersion>
-    <SystemReflectionTypeExtensionsVersion>4.3.0</SystemReflectionTypeExtensionsVersion>
-    <SystemResourcesResourceManagerVersion>4.3.0</SystemResourcesResourceManagerVersion>
-    <SystemRuntimeVersion>4.3.0</SystemRuntimeVersion>
-    <SystemRuntimeExtensionsVersion>4.3.0</SystemRuntimeExtensionsVersion>
-    <SystemRuntimeHandlesVersion>4.3.0</SystemRuntimeHandlesVersion>
-    <SystemRuntimeInteropServicesVersion>4.3.0</SystemRuntimeInteropServicesVersion>
-    <SystemRuntimeInteropServicesRuntimeInformationVersion>4.3.0</SystemRuntimeInteropServicesRuntimeInformationVersion>
-    <SystemRuntimeNumericsVersion>4.3.0</SystemRuntimeNumericsVersion>
-    <SystemRuntimeSerializationJsonVersion>4.3.0</SystemRuntimeSerializationJsonVersion>
-    <SystemRuntimeSerializationPrimitivesVersion>4.3.0</SystemRuntimeSerializationPrimitivesVersion>
-    <SystemSecurityAccessControlVersion>4.3.0</SystemSecurityAccessControlVersion>
-    <SystemSecurityCryptographyAlgorithmsVersion>4.3.0</SystemSecurityCryptographyAlgorithmsVersion>
-    <SystemSecurityCryptographyCspVersion>4.3.0</SystemSecurityCryptographyCspVersion>
-    <SystemSecurityCryptographyEncodingVersion>4.3.0</SystemSecurityCryptographyEncodingVersion>
-    <SystemSecurityCryptographyPrimitivesVersion>4.3.0</SystemSecurityCryptographyPrimitivesVersion>
-    <SystemSecurityCryptographyX509CertificatesVersion>4.3.0</SystemSecurityCryptographyX509CertificatesVersion>
-    <SystemSecurityPrincipalWindowsVersion>4.3.0</SystemSecurityPrincipalWindowsVersion>
-    <SystemTextEncodingVersion>4.3.0</SystemTextEncodingVersion>
     <SystemTextEncodingCodePagesVersion>4.3.0</SystemTextEncodingCodePagesVersion>
-    <SystemTextEncodingExtensionsVersion>4.3.0</SystemTextEncodingExtensionsVersion>
-    <SystemTextRegularExpressionsVersion>4.3.0</SystemTextRegularExpressionsVersion>
-    <SystemThreadingVersion>4.3.0</SystemThreadingVersion>
-    <SystemThreadingTasksVersion>4.3.0</SystemThreadingTasksVersion>
     <SystemThreadingTasksDataflowVersion>4.5.24</SystemThreadingTasksDataflowVersion>
+    <!-- We need SVT assembly version at least 4.0.3.0 on net47 to make F5 work against Dev15 - see https://github.com/dotnet/roslyn/issues/29705 -->
```

Comment:
Nitpick: don't abbreviate assembly names in comments; it makes it harder to grep for stuff later.

Reply:
Fixed.